firefox-ios/firefox-ios-tests/Tests/ClientTests/AppIconSelection/AppIconSelectionTelemetryTests.swift

```diff
@@ -0,0 +1,38 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Glean
+import XCTest
+
+@testable import Client
+
+final class AppIconSelectionTelemetryTests: XCTestCase {
+    // For telemetry extras
+    let nameIdentifierKey = "name"
+
+    override func setUp() {
+        super.setUp()
+        // Due to changes allow certain custom pings to implement their own opt-out
+        // independent of Glean, custom pings may need to be registered manually in
+        // tests in order to put them in a state in which they can collect data.
+        Glean.shared.registerPings(GleanMetrics.Pings.shared)
+        Glean.shared.resetGlean(clearStores: true)
```

Comment:
calling these from Glean caused some flaky tests in the past, any issues with using the mockGleanWrapper here?

Reply:
same for `SettingsTelemetryTests`